README.md

```diff
@@ -80,9 +80,13 @@ The documentation repository uses the following JavaScript tools to keep source
 
 To set these tools up:
 
-1. Install Node.js® from [nodejs.org][18]
-2. Open a command prompt, and change to the directory of your docs repository clone.
-3. Run `npm install`
+1. Ensure [Git][19] is installed and on `$PATH`.
+
+   The Husky installation may fail with an error message "Cannot read property 'toString' of null" if Git is not found on `$PATH`.
```

Comment:
Not sure how common this error message will be, and I'm usually reluctant to treat the exception like the rule, but let's add it for now, and maybe fold it into a collapsible Troubleshooting section later.